routes/user.js

```diff
@@ -76,28 +82,260 @@ router.post('/library', function(req, res, next) {
+            }
+        })
+    })
+})
+
+router.delete('/playlist/:playlist_name/export1', function(req, res) {
+    // var playlistID = ObjectId(req.body.playlistID)
+    // var songID = ObjectId(req.body.songID)
+    // var translationID = ObjectId(req.body.translationID)
+    var playlistID = req.body.playlistID
+    var songID = req.body.songID
+    var translationID = req.body.translationID
+    console.log(req.body)
+    ExportSong.findOne({
+            owner: playlistID,
+            song: songID
+        }, function(err, es) {
+            if (err) return handleError(err)
+            var index = es.translations.indexOf(translationID)
+            if (index > -1) {
+                es.translations.splice(index, 1)
+            }
+            es.save(function(err) {
+                if (err) {
+                    res.status(400).send('error ' + err)
+                } else {
+                    res.send({
+                        msg: 'deleting done'
+                    })
+                }
+            })
+        })
+        // ExportSong.update({_id: playlistID}, {
+        //     $pull: {
+        //         translations: translationID
+        //     }
+        // }, {
+        //     multi: true
+        // }, function(err) {
+        //     if (err) return handleError(err)
+        //     res.send({
+        //         msg: 'deleting done'
+        //     })
+        // })
+})
+
+router.get('/playlist/:playlist_name/export2', function(req, res) {
+    // var playlistName = req.params.playlist_name
+    // Playlist.find({owner: req.user._id, name: playlistName})
+    // // .populate(songs)
+    res.render('export2', {
+        playlistName: req.params.playlist_name
+    })
+})
+
+router.get('/playlist/:playlist_name/export3', function(req, res) {
+    var playlistName = req.params.playlist_name
+    var songss = []
+    var temp = []
+    Playlist.findOne({
+        owner: req.user._id,
+        name: playlistName
+    }, function(err, playlist) {
+        if (err) return handleError(err)
+        ExportSong.find({
+                owner: playlist._id
+            })
+            .populate('song translations')
+            .exec(function(err, songs) {
+                // console.log(songs)
+                if (err) return handleError(err)
+                songs.forEach((s, i, arr) => {
+                    temp = []
+                    temp.push(s.song)
+                    s.translations.forEach((t) => {
+                            temp.push(t)
+                        })
+                        // console.log(temp)
+                    songss.push(temp)
+                    if (i === arr.length - 1) {
+                        console.log(songss)
+                        res.render('export3', {
+                            songss: songss
+                        })
+                    }
+                })
+            })
+    })
+
+})
+
+router.post('/playlist/:playlist_name/export3', function(req, res) {
```

Comment:
same here and for export1, include comments explaining each export step and/or make the route name more descriptive.